specification/cognitiveservices/data-plane/LUIS/Runtime/v2.0/LUIS-Runtime.json

```diff
@@ -0,0 +1,353 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LUIS Endpoint API",
+    "version": "v2.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{AzureRegion}.api.cognitive.microsoft.com",
+    "parameters": [
+      {
+        "name": "AzureRegion",
+        "description": "Supported Azure regions for Cognitive Services endpoints.",
+        "x-ms-parameter-location": "client",
```

Comment:
For GA, what is the region list that should be included?
Or should we use the [BasicRegion](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/Common/BasicRegions.json) or [ExtendedRegion](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/Common/ExtendedRegions.json) sets?